src/drive/web/modules/drive/files.js

```diff
@@ -21,3 +23,14 @@ export const isReferencedByAlbum = file => {
   }
   return false
 }
+/**
+ *
+ * @param {object} file io.cozy.files
+ * @param {object} client CozyClient instance
+ * @return boolean If the file was created on my cozy
+ */
+export const isNoteMine = (file, client) => {
+  const myCozyUrl = client.getStackClient().uri
+  const noteCreatedOnCozy = get(file, 'cozyMetadata.createdOn')
+  return noteCreatedOnCozy.startsWith(myCozyUrl)
```

Comment:
A bit hacky but ok :) I guess it should be possible to see in the sharing data who the creator is.

Reply:
That said, `createdOn` isn't a standard metadata field? https://github.com/cozy/cozy-doctypes/tree/master/docs#document-metadata Are you sure it exists?

Reply:
https://github.com/cozy/cozy-doctypes/pull/125 ;)

Reply:
>A bit hacky but ok :) I guess it should be possible to see in the sharing data who the creator is.

Definitely hacky, will be changed in the next few days I think ;)

Reply:
https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.files.md#cozymetadata

It's specific for file